scripts/variantstore/scripts/convert_weight_file_CSV_to_bed.py

```diff
@@ -1,13 +1,16 @@
 import argparse
 import pandas as pd
+import csv
 
 location_offset = 1000000000000
+hg38_contig_map = {1: 'chr1', 2: 'chr2', 3: 'chr3', 4: 'chr4', 5: 'chr5', 6: 'chr6', 7: 'chr7', 8: 'chr8', 9: 'chr9', 10: 'chr10', 11: 'chr11', 12: 'chr12', 13: 'chr13', 14: 'chr14', 15: 'chr15', 16: 'chr16', 17: 'chr17', 18: 'chr18', 19: 'chr19', 20: 'chr20', 21: 'chr21', 22: 'chr22', 23: 'chrX', 24: 'chrY'}
```

Comment:
maybe we could build most of this with an iteration rather than risk a repeat of the last horribly embarrassing bug

Reply:
That's possible.  It's the x and y which are annoying, though.  

The main reason I chose to implement it this way (despite the issue we had last time) is that, inasmuch as our future method for handling arbitrary contigs involves passing in a map, it seemed consistent to store the default one as a map as well.  There's consistency in making all behavior just come down to choosing the right map.